README.md

```diff
@@ -23,23 +23,28 @@ The workflow will be set to fail if:
-| `dry-run`                  | Whether to submit an analysis (false/undefined) or only test the connection and credentials (true)                                                                                                | `undefined`   | No              |
+| Input Name                 | Description                                                                                                                                                                                       | Default Value | Required                       |
+|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|--------------------------------|
+| `server-url`               | The URL for the SRM server (typically ends with `/srm`)                                                                                                                                           |               | Yes                            |
+| `api-key`                  | An API Key or Personal Access Token to use when connecting to SRM                                                                                                                                 |               | Yes                            |
+| `project-id`               | The ID of a project (an integer) created in SRM                                                                                                                                                   | `undefined`   | Yes<sup>[1]</sup>              |
+| `project-name`             | The name of a project created in SRM                                                                                                                                                              | `undefined`   | Yes<sup>[1]</sup>              |
+| `auto-create-project`      | Whether or not SRM should automatically create a specified project name if it does not exist                                                                                                      | `false`       | No<sup>[2]</sup>               |
+| `base-branch-name`         | The parent branch name of a project created in SRM                                                                                                                                                | `undefined`   | No<sup>[2]</sup><sup>[3]</sup> |
+| `target-branch-name`       | The target branch name of a project created in SRM. <br/>SRM automatically creates the branch if it does not exist yet in the project, and the new branch will be created from `base-branch-name` | `undefined`   | No<sup>[2]</sup>               |
```

Comment:
Updated the superscript formatting in the Required column to be like `[n]`. Makes it look cleaner especially when multiple superscripts are involved imo.

![image](https://github.com/user-attachments/assets/bb6da45a-9c0d-41bb-97e8-330d0b4fddf8)